db/schema_tables.cc

```diff
@@ -695,16 +695,16 @@ redact_columns_for_missing_features(mutation m, schema_features features) {
  * Read schema from system keyspace and calculate MD5 digest of every row, resulting digest
  * will be converted into UUID which would act as content-based version of the schema.
  */
-future<utils::UUID> calculate_schema_digest(distributed<service::storage_proxy>& proxy, schema_features features)
+future<utils::UUID> calculate_schema_digest(distributed<service::storage_proxy>& proxy, schema_features features, noncopyable_function<bool(std::string_view)> accept_predicate)
```

Comment:
I'll merge, but the function signature for the predicate should have mentioned that it's the keyspace name being tested (either in the name of the predicate or the parameter).